### DISCUSSION TITLE
Distribution on Linux package managers

### DISCUSSION THREAD

**Discussion by TheOnlyTails** [JanDeDobbeleer/oh-my-posh#2487] [Ideas]
Would it be possible for oh-my-posh to be distributed on package managers other than homebrew? I don't really want to install another package manager on my system, and it'd be really nice if it was available on managers like apt, dnf, pacman, etc.

**Comment by JanDeDobbeleer** [comment]
@TheOnlyTails I looked into it and it's annoying to say the least. I'll provide an easy install script you can use instead.

**Reply by TheOnlyTails** [reply]
my main problem is that using the manual install command is tedious to say the least, especially when a new version comes out frequently (not that i'm complaining)

**Reply by TheOnlyTails** [reply]
maybe you could put it somewhere like flathub, so you won't need to publish it on a bunch of different package managers?

**Reply by JanDeDobbeleer** [reply]
@TheOnlyTails I understand the issue, I'll check flathub. But this one's low on my priority list due to time constraints.

**Reply by TheOnlyTails** [reply]
no problem, take all the time you need!

**Reply by TheOnlyTails** [reply]
from what i can tell from the docs, i think that if you have a simple script file it should be relatively easy to set up: https://docs.flatpak.org/en/latest/first-build.html

**Reply by JanDeDobbeleer** [reply]
I'm in the midst of a pretty challenging refactoring (#2489), then I'll try to finish the long awaited #767 and move to this one ;-)

**Reply by TheOnlyTails** [reply]
awesome, thank you!

**Comment by darthwalsh** [comment]
It seems like the AUR is setup to build oh-my-posh just fine. (I'm not an expert, but see below.)

Today on WSL Manjaro, I ran `yay -S oh-my-posh` (which prompted about installing build dependencies, and needed to install golang) but a few minutes later now I have working "oh-my-posh 19.18.1-1"!

Distributing binaries on package managers is probably extra work, but the build-from-source experience was just fine for me. I think it would be good to add tabs on https://ohmyposh.dev/docs/installation/linux for any package managers that are working.

**Reply by JanDeDobbeleer** [reply]
@darthwalsh the problem is that I'm not in control when it comes to those. So I'm always a bit reluctant to add them to the official docs.
